OPEN_SOURCE_POLICY.md

```diff
@@ -0,0 +1,163 @@
+transparent environment.
+
+This policy exists to support and protect the Elixir community. It aims to
+balance openness, collaboration, and respect for all contributors’ rights,
+ensuring that Elixir remains a trusted and innovative open source project.
+
+## 2. Scope
+
+This policy applies to the Elixir Programming language, located at
+https://github.com/elixir-lang/elixir. It covers every file, and contribution
+made, including documentation and any associated assets.
+
+## 3. Licensing
+
+All code released by the Elixir team is licensed under the
+[Apache-2.0](./LICENSES/Apache-2.0.txt) license. Additionally, the following
+licenses are recognized as permissible in this project:
+
+- The Unicode license, as documented at
+  [LicenseRef-scancode-unicode](./LICENSES/LicenseRef-scancode-unicode.txt)
+- The Elixir Trademark Policy, as documented at
+  [LicenseRef-elixir-trademark-policy](./LICENSES/LicenseRef-elixir-trademark-policy.txt)
+
+These licenses are considered acceptable for any files or code that form part of
+an Elixir repository. If a contribution requires a different license, it must
+either be rejected or prompt an update to this policy.
+
+## 4. Contributing to Elixir Projects
+
+Any code contributed to Elixir repositories must fall under one of the accepted
+licenses (Apache-2.0, Unicode, or Elixir Trademark). Contributions under any
+other license will be rejected unless this policy is formally revised to include
+that license. All files except those specifically exempted (e.g., certain test
+fixture files) must contain SPDX license and copyright headers
+(`SPDX-License-Identifier` and `SPDX-FileCopyrightText`). If a file qualifies
+for an exception, this must be configured in the ORT (Open Source Review Toolkit)
+configuration and undergo review.
+
+Contributions must not introduce executable binary files into the codebase.
+
+Every Elixir project within the organization will have an automated GitHub
+Action to enforce these rules. This mechanism aids in detecting non-compliant
+licenses or files early in the review process.
+
+## 5. Preservation of Copyright and License Information
+
+Any third-party code incorporated into Elixir projects must retain original
+copyright and license headers. If no such headers exist in the source, they must
+be added. This practice ensures that original authors receive proper credit and
+that the licensing lineage is preserved.
+
+## 6. Objectives
+
+The Elixir project aims to promote a culture of responsible open source usage.
+Specifically, our objectives include:
+
+### 6.1 Clearly Define and Communicate Licensing & Compliance Policies
+
+We will identify and document all third-party dependencies, ensure that license
+information is communicated clearly, and maintain a project-wide license policy
+or compliance handbook.
+
+### 6.2 Implement Clear Processes for Reviewing Contributions
+
+We will provide well-defined contribution guidelines. We utilize adopt a
```

Comment:
I think there are two verbs in this sentence?

Reply:
@sabiwara Oops, good find. I replaced it with

> We implement the Developer Certificate of Origin (DCO) for additional clarity regarding contributor rights and obligations.